packages/opentelemetry-plugin-ioredis/src/ioredis.ts

```diff
@@ -23,7 +23,7 @@ import { VERSION } from './version';
 export class IORedisPlugin extends BasePlugin<typeof ioredisTypes> {
   static readonly COMPONENT = 'ioredis';
   static readonly DB_TYPE = 'redis';
-  readonly supportedVersions = ['^2.0.0'];
+  readonly supportedVersions = ['>1 <5'];
```

Comment:
Do we support version 1.x.x? This version string will match that I believe? I think you're looking for `'>=2 <5'`

Reply:
According to https://semver.npmjs.com/ it's fine 🤷‍♂